templatefield/test/models.py

```diff
@@ -4,7 +4,11 @@
 
 
 class TemplatedText(models.Model):
-    value = fields.TemplateTextField()
+    value = fields.TemplateTextField(null=True)
```

Comment:
@percyperez I think it needs to be `null=True`, I did that to test this line: https://github.com/orcasgit/django-template-field/blob/master/templatefield/fields.py#L10